utils/RapidproUtils.ts

```diff
@@ -0,0 +1,217 @@
+import { IHttp, IHttpResponse, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+
+export class RapidproUtils {
+    private readonly MESSAGE_CHUNK_SIZE: number = 640;
+
+    private read: IRead;
+    private http: IHttp;
+    private authToken: string;
+    private baseUrl: string;
+    private closeTicket: string;
+
+    public constructor(read: IRead, http: IHttp, baseUrl: string, authToken: string, closeTicket: string) {
+        this.read = read;
+        this.http = http;
+        this.baseUrl = baseUrl;
+        this.authToken = authToken;
+        this.closeTicket = closeTicket;
+    }
+
+    public getAuthorizationHeader() {
+        const auth = this.authToken;
+        return {Authorization: `Token ${auth}`};
+    }
+    public getCloseTicket() {
+        return this.closeTicket;
+    }
+
+    public async getLogMessages(token: string, after: string): Promise<any> {
+
+        const messagesResponse = await this.getContactMessages(token, after);
+        const messages = messagesResponse.data.results.reverse();
```

Comment:
That `results` can be so big, right? So, you can iterates that array with a common for starting from the last index to 0.

Reply:
Instead of to use `reverse`.